gradle/libs.versions.toml

```diff
@@ -20,7 +20,7 @@ qrose = "1.0.1"
 recaptcha = "18.6.1"
 sentry = "4.12.0"
 serialization = "1.7.3"
-swisstransfer = "0.9.0"
+swisstransfer = "0.9.2"
```

Comment:
Maybe we should do this bump in its own PR, before this one ?

Reply:
![image](https://github.com/user-attachments/assets/b0a24f8a-136b-4208-8bc4-7f92ac4905c0)